include/proxsuite/proxqp/sparse/solver.hpp

```diff
@@ -844,7 +844,7 @@ qp_solve(Results<T>& results,
           is_dual_feasible(dual_feasibility_lhs)) {
         if (settings.check_duality_gap) {
           if (std::abs(results.info.duality_gap) <=
```

Comment:
We use both ``std::abs`` and ``std::fabs`` in the four instances of this check.

AFAIU the latter is float-specific and the former more general, so we should consistently use ``std::fabs``, right?

Reply:
Yes, this is a mistake ...

Reply:
OK, done in 2d7a82bae402a2fda422fe2f671c56c1a140c89e.